pydantic/main.py

```diff
@@ -853,7 +853,9 @@ def __eq__(self, other: Any) -> bool:
             return self.dict() == other
 
     def __repr_args__(self) -> 'ReprArgs':
-        return [(k, v) for k, v in self.__dict__.items() if self.__fields__[k].field_info.repr]
+        return [
+            (k, v) for k, v in self.__dict__.items() if self.__fields__.get(k) and self.__fields__[k].field_info.repr
```

Comment:
```suggestion
            (k, v) for k, v in self.__dict__.items() if k not in self.__fields__ or self.__fields__[k].field_info.repr
```

We **should** included these properties in `repr()`